documentation/src/docs/admin-guide/lts-sizing.md

```diff
@@ -106,4 +110,47 @@ the thread pool size to 2-4 threads.
 - _Maximum size of single chunk in LTS: In production, it may be required to revisit the value of `writer.rollover.size.bytes.max`.
 Limiting the size of chunks in LTS poses a trade-off: if the value is low, the system will achieve fine retention granularity 
 (good) but extra metadata overhead (not so good). On the contrary, a high value in this parameter induces coarse retention 
-granularity with less metadata overhead. Limiting chunk sizes between 1GB-16GB seems reasonable in many scenarios. 
\ No newline at end of file
+granularity with less metadata overhead. Limiting chunk sizes between 1GB-16GB seems reasonable in many scenarios.
+
+## Right-Sizing Tier-1 Storage (Bookkeeper)
+
+Pravega makes use of tier-1 storage to store durably and with low latency events until they are moved to long-term storage. 
+Thus, we need guidelines to right-sizing the storage capacity required in tier-1 (i.e., Bookkeeper).
+
+To this end, there are three variables that we need to take into account:
+- _Max ledger size (`bookkeeper.ledger.size.max`)_: Maximum Ledger size (bytes) in BookKeeper. Once a Ledger reaches this size, 
+it will be closed and another one open. _Type_: `Long`. _Default_: `1073741824`. _Update-mode_: `per-server`.
```

Comment:
'another one open' -> 'another one opened'

Suggestion: put "(1 GiB)" after default value in bytes.

Reply:
Fixed.